### DISCUSSION TITLE
update version in winget

### DISCUSSION THREAD

**Discussion by Steve4524** [OpenSC/OpenSC#3261] [General]
The version of OpenSC on WinGet is rather old, are there any plans to publish a newer one?
```
PS C:\> winget search opensc.opensc --versions
Found OpenSC smartcard framework [OpenSC.OpenSC]
Version
--------
0.23.0.0
```
Thanks.   -Steve

**Comment by Jakuje** [comment]
There were issues and PRs to update OpenSC there, but they were not finalized. The original author who asked to add the package in there was not involved after the first release in there.

https://github.com/microsoft/winget-pkgs/issues/147910
https://github.com/microsoft/winget-pkgs/pull/147916

If you are winget user and would like to get involved, updating the package in there would be a good start. From the PR it sounds like it is not very complicated.

I am not using winget nor windows and updating all distros and package managers would be a full time job for one.

Let me know if there is something I can get help you though.
